SignalUITests/SignalUITests.swift

```diff
@@ -0,0 +1,737 @@
+//
+//  SignalUITests.swift
+//  SignalUITests
+//
+//  Created by Matthew Kotila on 1/13/16.
+//  Copyright © 2016 Open Whisper Systems. All rights reserved.
+//
+
+import XCTest
+
+class SignalUITests: XCTestCase {
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        XCUIApplication().launch()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+    }
+    
+    // requires unverified app
```

Comment:
Hi Michael, I'm not extremely versed in UI tests, so when I encountered the problem of "application state" (i.e. testing UI pre-verification and post-verification), I simply wrote tests for an app that was pre-verified, and then verified it manually using the iOS simulator with valid information, and then wrote tests for the post-verified app. I'm unsure if there's a way to automatically input information (such as contacts, verification codes, etc.) once and start a different set of UI tests intended for a verified app. Additionally, I'm unsure if there's a way to automatically "wipe" an app in order to test pre-verification UI. All very interesting questions in the realm of UI testing!